src/check/validators/test_names.rs

```diff
@@ -128,18 +153,21 @@ mod tests {
 
         let disallowed_names = [
             "test",
+            "test123_Description",
             "testDescription",
             "testDescriptionMoreInfo",
-            // TODO The below are tough to prevent without regex look-ahead support.
-            // "test_RevertIfCondition",
-            // "test_RevertWhenCondition",
-            // "test_RevertOnCondition",
-            // "testFuzz_RevertIfDescription",
-            // "testFuzz_RevertWhenDescription",
-            // "testFuzz_RevertOnDescription",
-            // "testForkFuzz_RevertIfCondition",
-            // "testForkFuzz_RevertWhenCondition",
-            // "testForkFuzz_RevertOnCondition",
+            "testRevertIfCondition",
+            "testRevertIf_Condition",
+            "test_RevertIfCondition",
+            "test_RevertWhenCondition",
+            "test_RevertOnCondition",
+            "testFuzz_RevertIfDescription",
+            "testFuzz_RevertWhenDescription",
+            "testFuzz_RevertGivenDescription",
+            "testFuzz_RevertOnDescription",
+            "testForkFuzz_RevertIfCondition",
+            "testForkFuzz_RevertWhenCondition",
+            "testForkFuzz_RevertOnCondition",
```

Comment:
Shouldn't `testForkFuzz_RevertGivenCondition` be added here?

Reply:
And maybe `test_RevertGivenCondition`?